tests/test_georaster.py

```diff
@@ -1291,26 +1291,105 @@ def test_interp(self) -> None:
         # Out[34]: (75.0, 302.0)
 
     def test_value_at_coords(self) -> None:
+        """
+        Test that value at coords works as intended
+        """
 
-        r = gr.Raster(self.landsat_b4_path)
-        r2 = gr.Raster(self.landsat_b4_crop_path)
-        r.crop(r2)
+        # -- Tests 1: check based on indexed values --
+
+        # Open two rasters and crop the first one
```

Comment:
Looks like the comment is not appropriate anymore?

Reply:
Updated